packages/core/admin/ee/admin/content-manager/pages/ListSettingsView/constants.js

```diff
@@ -1,7 +1,7 @@
-export const REVIEW_WORKFLOW_STAGE_SORT_OPTION_NAME = {
-  value: 'strapi_stage[name]',
+export const REVIEW_WORKFLOW_ASSIGNEE_SORT_OPTION_NAME = {
```

Comment:
Am I missing something - why would you want to overwrite the stage sorting? Don't you want to add a new option?

Reply:
Fixed & rebeased